library/src/main/kotlin/com/a99/rxplaces/RxTextView.kt

```diff
@@ -0,0 +1,54 @@
+package com.a99.rxplaces
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.TextView
+import rx.Observable
+import rx.Subscriber
+import rx.android.MainThreadSubscription
+import rx.android.MainThreadSubscription.verifyMainThread
+
+class RxTextView {
```

Comment:
I believe that RxChanges is too broad. This class does only one thing today, which is good, and I do believe that the actual name is better than `RxChanges`.